### ISSUE TITLE
Layout broken after installation of 1.5.2

### ISSUE THREAD

**Issue by computersalat** [webkul/hotelcommerce#333]
After installation of 1.5.2 the layout is completely broken.
Looks like the theme is not working as expected.
I Found one error and fixed it: https://github.com/computersalat/hotelcommerce/commit/9d028d70c3d816328c654df14c2ef48999684eb5

I also found a lot of code changes `back to PrestaShop`: https://github.com/computersalat/hotelcommerce/commit/b2f6dd693c09656da7df63f24c70addd6ccd4708

But the front-end still looks devastated.
Someone needs to do a test install and verify

**Comment by sumitwebkul** [comment]
Regarding layout issues, Please send the screenshots of the layout issues. So that we can know the exact issues.

Regarding Prestashop text in the emails, We already working on this issue and will resolve this in future upgrade.

Thanks for your contributions.

**Comment by computersalat** [comment]
![20220317_webkul_hotelcommerce_i333](https://user-images.githubusercontent.com/161880/158853278-b5705459-4c65-47b5-b92c-5ed7856c59f4.jpg)
Please find attached

**Comment by sumitwebkul** [comment]
We do not find any such issue on QloApps installation.
Please let us know if there were some issue while installation or any specific condition for this issue?
Also, please let us know your PHP version and MySQL version.
Thanks

**Comment by computersalat** [comment]
> We do not find any such issue on QloApps installation. Please let us know if there were some issue while installation or any specific condition for this issue? Also, please let us know your PHP version and MySQL version. Thanks

Sorry, but you must see at least the font loading problem ...
themes/hotel-reservation-theme/header.tpl: line 76
```
-		<link href='//fonts.googleapis.com/css?family=Oxygen:400,300,700' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Oxygen:400,300,700' rel='stylesheet' type='text/css'>
```
There was no issue while installing.
I just needed to switch on both `SSL` switches in the back-end, because the site is completely secured by SSL.

php --version
```
PHP 7.4.6 (cli) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with the ionCube PHP Loader + ionCube24 v10.4.5, Copyright (c) 2002-2020, by ionCube Ltd.
    with Zend OPcache v7.4.6, Copyright (c), by Zend Technologies
```
mysql --version
```
mysql  Ver 15.1 Distrib 10.5.15-MariaDB, for Linux (x86_64) using  EditLine wrapper
```

**Comment by computersalat** [comment]
@sumitwebkul 
After fixing ( https://github.com/computersalat/hotelcommerce/commit/1edafa77e51cdb102a97bde35d49966af9f2835e ) all of your `old` .htaccess files it is working now.
You should cherry-pick my commits

**Comment by sumitwebkul** [comment]
Yes, you have to enable both SSL options if ssl is enabled on your domain.
regarding your fixes. we will look into it and update you on this.
Thanks

**Comment by shreesh-webkul** [comment]
We are unable to recreate the above issue at our end.
Can you please provide us with a link where we can check this issue?

**Comment by computersalat** [comment]
> We are unable to recreate the above issue at our end. Can you please provide us with a link where we can check this issue?

what webserver are you using ?
use `apache >=` **2.4** then you will see that all your `.htaccess` are written for `apache <` **2.4**

Why are you not using my provided fix in #335 ?

**Comment by shreesh-webkul** [comment]
We are checking the PR, Will merge once we have checked the issue.

**Comment by qfyra** [comment]
I get the exact same issue as @computersalat using v 1.5.2.0. Running PHP 7.4 and Apache 2.4. No issues during installation. Also had to set "Enable SSL" and "Enable SSL on all pages" to YES, but other than that I get the exact same broken look as the screenshot provided by @computersalat. After applying the .htaccess fixes provided by @computersalat, site seems to be working fine.

**Comment by shreesh-webkul** [comment]
Thanks for raising this issue and helping us improve QloApps.

We are checking the issue. 
Last time we tried multiple ways but was unable to recreate this issue. Will let you know if we need anything from your side.

**Comment by shreesh-webkul** [comment]
@qfyra @computersalat, can you please provide your apache2.conf file.

**Comment by computersalat** [comment]
@shreesh-webkul 
simply install an openSUSE 15.3 with apache. then install QloApps 1.5.2.
All your .htaccess files are written for `OLD` apache 2.2. They do not work with apache 2.4.
That is the problem.
What do you think you can find in the apache conf ?
Just take the fix in #335 and you are fine. That fix will nicely work even with `OLD` apache.

**Comment by shreesh-webkul** [comment]
@computersalat @qfyra  We were able to recreate the given issue in openSUSE 15.3.
PR #335 resolve the given issue, but it requires some changes so that we can merge the PR.
Please update the PR.

**Comment by pawelkierklo** [comment]
Have a same problem (1st screen). I found problem, please resolve.

![image](https://user-images.githubusercontent.com/26326399/225910050-8d778640-791e-4da7-8a6b-e7e920ff4506.png)

**Comment by computersalat** [comment]
@shreesh-webkul 
> @computersalat @qfyra We were able to recreate the given issue in openSUSE 15.3. PR #335 resolve the given issue, but it requires some changes so that we can merge the PR. Please update the PR.

Your ChangeRequest is completely wrong.
